command-line/command-line-advanced.md

```diff
@@ -41,6 +41,7 @@ Teach people some handier tools for advanced file searching and processing
 - [Filenames and Pathnames in Shell: How to do it Correctly](https://www.dwheeler.com/essays/filenames-in-shell.html)
 - [An Awk Primer/Awk Command-Line Examples](https://en.wikibooks.org/wiki/An_Awk_Primer/Awk_Command-Line_Examples)
 - [I/O Redirection](http://wiki.bash-hackers.org/syntax/redirection)
+- [Videos](https://www.youtube.com/playlist?list=PL6gx4Cwl9DGCkg2uj3PxUWhMDuTw3VKjM)
```

Comment:
@Soumya0803 this link just takes me to a channel. Please provide a link to 1 or 2 videos that demonstrate the specific skills being taught in this topic’s outline and slideshow.

Reply:
@alodahl I found a playlist which consists of 14 videos on command line.
Link: https://www.youtube.com/watch?v=bE9DyH43C2I&list=PLVqGqrTs4ZWOhcApSWYIX_rnPMZDAClJa&index
Please review this link. 
I am also trying,if I could find 1 or 2 videos which covers the skills being taught.

Reply:
The materials section is required, and i can’t ask them to watch 1.5 hours of video. The other option is to do a screen recording and email it to techtonica@gmail.org, and send us the link in the comments here to review.

Reply:
You can put these videos under “supplemental materials”; we just can’t require 14 videos.